src/creamy-spaghetti-aglio-e-olio.md

```diff
@@ -0,0 +1,40 @@
+# Creamy Spaghetti Aglio e Olio
+
+![Creamy Spaghetti Aglio e Olio](pix/creamy-spaghetti-aglio-e-olio.webp)
+
+Took the classic aglio e olio spaghetti recipe and added some eggs and parmesan cheese.
+
+- 🍳 Cook time: 15 min
+- 🍽️ Servings: 1
+
+## Ingredients
+
+- 57 grams of Linguine spaghetti
+- 1/4 cup extra virgin olive oil
```

Comment:
```suggestion
- 1/4 cup (60ml) extra virgin olive oil
```

Reply:
I meant ml not mg. May you please add that as such and push the commit, as clicking resolved on github doesn't do anything but collapse the box showing the suggestion btw. Some people think pressing resolve enacts the change, which I assume is also you because you thumbed up my suggestion but didn't change it with a new commit. But yeah, I meant 60ml rounded not 58grams of olive oil anyway lul.

Reply:
Ah right, did not know that. Pushed your suggestion now :)